.github/actions/kamel-config-cluster-kind/action.yml

```diff
@@ -23,11 +23,12 @@ runs:
   steps:
     - id: install-cluster
       name: Install Cluster
-      uses: container-tools/kind-action@v1
+      uses: container-tools/kind-action@61f1afd4807b0dac84f3232ec99e45c63701d220
```

Comment:
We can merge with this commit id, but, please, create a follow up issue to release the kind-action and update here accordingly.

Reply:
Here it is https://github.com/apache/camel-k/issues/4063